src/async-components/views/dialogs/secretstorage/CreateSecretStorageDialog.js

```diff
@@ -268,6 +277,22 @@ export default class CreateSecretStorageDialog extends React.PureComponent {
         return this.state.zxcvbnResult && this.state.zxcvbnResult.score >= PASSWORD_MIN_SCORE;
     }
 
+    _renderPhaseRestoreKeyBackup() {
+        const DialogButtons = sdk.getComponent('views.elements.DialogButtons');
+        return <div>
+            <p>{_t(
+                "Key Backup is enabled on your account but has not been set " +
+                "up from this sign-in. To set up secret storage, " +
+                "restore your key backup.",
+            )}</p>
+            <DialogButtons primaryButton={_t('Cancel')}
```

Comment:
Would it make sense to add a positive action to open the restore key backup screen, instead of cancel and go find another button in settings?

Reply:
Yeah, probably - done. Only slightly annoying thing is that it then doesn't throw you back to the cross-signing bootstrap flow once you're done restoring.

Reply:
Ah yeah. Presumably something we can tweak, but hopefully this will be an edge case anyway...